elements/pfe-button/src/pfe-button.scss

```diff
@@ -0,0 +1,114 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: button;
+
+$LOCAL-VARIABLES: (
+  BackgroundColor: pfe-var(ui-accent),
+  BackgroundColor--hover: pfe-var(ui-accent--hover),
+  Color: pfe-var(ui-base--text),
+  FontSize: pfe-var(font-size),
+  Border: 0,
+  BorderColor--after: transparent,
+  BorderRadius: pfe-var(surface--border-radius),
+  Padding: calc(#{pfe-var(container-padding)} / 2) pfe-var(container-padding),
+  LineHeight: pfe-var(line-height),
+);
+
+// Nested internal variables (pfe-local calls), maps cannot "self-reference"
+$LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, (
+  Border--after: pfe-var(ui--border-width) pfe-var(ui--border-style) pfe-local(BorderColor--after),
+  Border--after--hover: pfe-var(ui--border-width) pfe-var(ui--border-style) pfe-local(BorderColor--after),
+));
+
+:host {
+  display: inline-block;
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+:host([pfe-variant="primary"]) button,
+button {
+  background-color: pfe-local(BackgroundColor);
+  color: pfe-local(Color);
+  font-size: pfe-local(FontSize);
+  font-family: pfe-var(font-family, $fallback: inherit);
```

Comment:
Font family will inherit - do we need to set this on the button specifically?

Reply:
I think we still need this for buttons in forms

Reply:
Right but it will still inherit, even if it's in a form.

Reply:
I don't think buttons are not set to inherit font family by default. I'm pro-this CSS
![image](https://user-images.githubusercontent.com/5607236/90673452-c41b1d00-e225-11ea-9b4d-b690d4da3a6e.png)

Reply:
I hate that about the spec :angry: